pkgs/development/libraries/openexr/3.nix

```diff
@@ -20,6 +20,12 @@ stdenv.mkDerivation rec {
 
   outputs = [ "bin" "dev" "out" "doc" ];
 
+  patches =
+    # Disable broken test on musl libc
+    # https://github.com/AcademySoftwareFoundation/openexr/issues/1556
+    lib.optional stdenv.hostPlatform.isMusl ./disable-iex-test.patch
+  ;
```

Comment:
We shouldn't do conditional patches if avoidable and instead disable the test in preCheck if possible.